### PR TITLE
Forms Automated Test - LPS-133878 | master

### PR DESCRIPTION
**Pull request by wesleyroberts**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/Form.macro [hunk 1]
```diff
@@ -4,6 +4,14 @@ definition {
 	var formName = "Created Form Name";
 	var rowNumber = "1";
 
+	macro addFieldToFieldsGroup {
+		DragAndDrop.javaScriptDragAndDropToBottom(
+			key_fieldLabel = "${targetFieldName}",
+			key_fieldName = "${sourceFieldLabel}",
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/Form.macro:10]: @wesleyroberts, we're assigning a field name to a field label and vice-versa, can you please make it consistent?

I mean:
```
key_fieldLabel = "${targetFieldLabel}",
key_fieldName = "${sourceFieldName}",
```

**Comment by natocesarrego** [comment]
@wesleyroberts, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/macros/Form.macro (hunk 1 above)
Comment (wesleyroberts) [reply portal-web/test/functional/com/liferay/portalweb/macros/Form.macro]: changes made

**Comment by wesleyroberts** [comment]
@natocesarrego  rebase made .

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsGroup.testcase [hunk 2]
```diff
@@ -122,14 +122,54 @@ definition {
...
+
+		FormFields.editHelpTextField(fieldHelpText = "Test Help Text Numeric");
+
+		Form.addFieldToFieldsGroup(
+			sourceFieldName = "Numeric",
+			targetFieldLabel = "Text");
+
+		FormFields.viewTextLabel(fieldLabel = "Fields Group");
+
+		Form.viewFormPlaceholderText(key_value = "Test Placeholder Text");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsGroup.testcase:160]: @wesleyroberts, can we use the macro `FormFields.viewFieldPlaceholderText` instead?

Almost there, @wesleyroberts. :slightly_smiling_face:

Comment (wesleyroberts) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsGroup.testcase]: yes, we can use this macro  FormFields.viewFieldPlaceholderText, change made @natocesarrego

**Comment by natocesarrego** [comment]
@wesleyroberts, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

This pull is ready to go. :slightly_smiling_face: 

Thanks.

**Comment by wesleyroberts** [comment]
I resolved the conflicts and the SF changes some file, this is a problem ? @natocesarrego

**Comment by natocesarrego** [comment]
Let's move on, @wesleyroberts, probably it was introduced by updates on the SF code.

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**